packages/tools/locales/vi.po

```diff
@@ -4147,223 +4159,216 @@ msgstr "Phiên bản thông tin: %s"
+msgstr "Đính kèm bất kỳ tập tin nào"
 
-#~ msgid "Unknown log level: %s"
-#~ msgstr "Không tìm thấy log level: %s"
+msgid "Unknown log level: %s"
+msgstr "Không tìm thấy log level: %s"
 
-#~ msgid "Unknown level ID: %s"
-#~ msgstr "Không tìm thấy level ID: %s"
+msgid "Unknown level ID: %s"
+msgstr "Không tìm thấy level ID: %s"
 
-#, fuzzy
-#~ msgid "Synchronize"
-#~ msgstr "Đồng bộ hóa"
+msgid "Synchronize"
+msgstr "Đồng bộ"
 
-#~ msgid "Json Export Directory"
-#~ msgstr "Json Export Directory"
+msgid "Json Export Directory"
+msgstr "Json Export Directory"
 
-#~ msgid ""
-#~ "This item is currently encrypted: %s \"%s\". Please wait for all items to "
-#~ "be decrypted and try again."
-#~ msgstr ""
-#~ "Mục hiện tại đang được mã hóa: %s \"%s\". Vui lòng chờ cho tất cả các mục "
-#~ "được giải mã và thử lại."
+msgid ""
+"This item is currently encrypted: %s \"%s\". Please wait for all items to be "
+"decrypted and try again."
+msgstr ""
+"Mục hiện tại đang được mã hóa: %s \"%s\". Vui lòng chờ cho tất cả các mục "
+"được giải mã và thử lại."
 
-#, fuzzy
-#~ msgid "Remove tag \"%s\" and its descendant tags from all notes?"
-#~ msgstr "Gỡ bỏ tag \"%s\" khỏi tất cả các notes?"
+msgid "Remove tag \"%s\" and its descendant tags from all notes?"
+msgstr "Gỡ bỏ nhãn \"%s\" khỏi tất cả các ghi chú?"
 
-#, fuzzy
-#~ msgid ""
-#~ "Could not synchronise with OneDrive.\n"
-#~ "\n"
-#~ "This error often happens when using OneDrive for Business, which "
-#~ "unfortunately cannot be supported.\n"
-#~ "\n"
-#~ "Please consider using a regular OneDrive account."
-#~ msgstr ""
-#~ "Không thể đồng bộ hóa với OneDrive.\n"
-#~ "\n"
-#~ "Lỗi này thường gặp phải khi sử dụng OneDrive cho doanh nghiệp, và không "
-#~ "được hỗ trợ.\n"
-#~ "\n"
-#~ "Vui lòng sử dụng tài khoản OneDrive phổ thông."
+msgid ""
+"Could not synchronise with OneDrive.\n"
+"\n"
+"This error often happens when using OneDrive for Business, which "
+"unfortunately cannot be supported.\n"
+"\n"
+"Please consider using a regular OneDrive account."
+msgstr ""
+"Không thể đồng bộ hóa với OneDrive.\n"
+"\n"
+"Lỗi này thường gặp phải khi sử dụng OneDrive cho doanh nghiệp, rất tiếc là "
+"loại tài khoản này chưa được hỗ trợ.\n"
+"\n"
+"Vui lòng sử dụng tài khoản OneDrive thông thường."
 
-#, fuzzy
-#~ msgid "Cannot move tag to this location."
-#~ msgstr "Không thể di chuyển notebook tới vị trí này"
+msgid "Cannot move tag to this location."
+msgstr "Không thể di chuyển nhãn tới vị trí này."
 
-#~ msgid "Add or remove tags"
-#~ msgstr "Gắn hoặc bỏ tags"
+msgid "Add or remove tags"
+msgstr "Gắn hoặc bỏ tags"
 
-#~ msgid "Link"
-#~ msgstr "Đường link"
+msgid "Link"
+msgstr "Đường link"
 
-#~ msgid "Content properties"
-#~ msgstr "Đặc tính của nội dung"
+msgid "Content properties"
+msgstr "Đặc tính của nội dung"
 
-#, fuzzy
-#~ msgid "Split"
-#~ msgstr "Chia đôi màn hình"
+msgid "Split"
+msgstr "Chia màn hình"
 
-#~ msgid "Resources"
-#~ msgstr "Tài nguyên"
+msgid "Resources"
+msgstr "Tài nguyên"
```

Comment:
How did these get uncommented? They should not even show up in Poedit.
These are most likely old strings.

Reply:
I actually have no idea about these strings. I've just translated anything that shows up in Poedit.

Reply:
Interesting. The file seems ok and these strings are not old translations.